scripts/src/create.ts

```diff
@@ -10,7 +10,7 @@ import { Order } from "./models/orders";
 
 import { init as initModels } from "./models";
 import { getConfig } from "./config";
-import { animalPoll, kikPoll, Poll } from "./services/offer_contents";
+import { animalPoll, kikPoll, tutorial, Poll, Tutorial } from "./services/offer_contents";
```

Comment:
optimize imports so `tutorial` would disappear

Reply:
`tutorial` is the data, the instance, and `Tutorial` is the type.

Reply:
I change `tutorial` to `getStartedTutorial` if it helps